lib/ex_unit/lib/ex_unit/case.ex

```diff
@@ -324,9 +324,10 @@ defmodule ExUnit.Case do
         end
       end
 
-  When using Mix, you can run all tests in a describe block as:
+  When using Mix, you can run all tests in a describe block (starting at line 123) as either:
 
       mix test --only describe:"String.capitalize/1"
+      mix test path/to/file:123
```

Comment:
I think we need to clarify this requires the *exact* line. So maybe we should rephrase this to:

```
When using Mix, you can run all tests in a describe block by name:

    mix test --only describe:"String.capitalize/1"

or by passing the exact line the describe block starts:

    mix test path/to/file:123
```

Thoughts?

Reply:
I agree, it's more detailed and clear this way.